geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/analytics/HistogramJob.scala

```diff
@@ -19,7 +19,9 @@ package org.locationtech.geomesa.jobs.analytics
 import com.twitter.algebird.Aggregator
 import com.twitter.scalding._
 import com.twitter.scalding.typed.UnsortedGrouped
+import org.apache.accumulo.core.data.{Mutation, Range => AcRange}
```

Comment:
(picky) We've mapped Range to AccRange and ARange.  It might be worth standardizing.